README.md

```diff
@@ -138,6 +138,28 @@ Class: `o-table--vertical-lines`, Mixin : `oTableVerticalLines`
 
 Thin lines will be rendered to the left and right of each `td` element giving the appearance of lines between columns.
 
+## Responsive
+
+In the fifth version of `o-table`, there are responsive options that can be used for displaying data in a table. 
```

Comment:
you don´t need to specify the version, as you will only see the readme if you´re looking at the correct release